src/commands/logs/list.rs

```diff
@@ -0,0 +1,119 @@
+use anyhow::Result;
+use clap::Args;
+
+use crate::api::{Api, Dataset, FetchEventsOptions};
+use crate::config::Config;
+use crate::utils::formatting::Table;
+
+/// Fields to fetch from the logs API
+const LOG_FIELDS: &[&str] = &[
+    "sentry.item_id",
+    "trace",
+    "severity",
+    "timestamp",
+    "message",
+];
+
+/// Arguments for listing logs
+#[derive(Args)]
+pub(super) struct ListLogsArgs {
+    #[arg(short = 'o', long = "org")]
+    #[arg(help = "The organization ID or slug.")]
+    org: Option<String>,
+
+    #[arg(short = 'p', long = "project")]
+    #[arg(help = "The project ID (slug not supported).")]
+    project: Option<String>,
+
+    #[arg(long = "max-rows", default_value = "100")]
+    #[arg(help = "Maximum number of log entries to fetch and display (max 1000).")]
+    max_rows: usize,
+
+    #[arg(long = "query", default_value = "")]
+    #[arg(help = "Query to filter logs. Example: \"level:error\"")]
+    query: String,
+}
+
+pub(super) fn execute(args: ListLogsArgs) -> Result<()> {
+    let config = Config::current();
+    let (default_org, default_project) = config.get_org_and_project_defaults();
+
+    let org = args
+        .org
+        .as_ref()
+        .or(default_org.as_ref())
+        .ok_or_else(|| {
+            anyhow::anyhow!("No organization specified. Use --org or set a default in config.")
```

Comment:
I don't think many users would know how to set a default in the config. I would prefer we highlight only the `--org` argument here

```suggestion
            anyhow::anyhow!("No organization specified. Please specify an organization using the --org argument.")
```

Reply:
we could specify how they could do that, like `(see sentry-cli config --help) to learn how to configure your cli`